src/routes/api/multiplayer/[...path]/+server.ts

```diff
@@ -7,14 +8,17 @@ export const fallback: RequestHandler = async ({ request, params, url, fetch })
 
   const searchParams = url.searchParams.toString();
 
-  const response = await fetch(`${MULTIPLAYER_API_URL}/${params.path}?${searchParams}`, {
-    method: request.method,
-    body: body || undefined,
-    headers: {
-      Authorization: `Bearer ${MULTIPLAYER_API_ACCESS_TOKEN}`,
-      'Content-Type': 'application/json',
+  const response = await fetch(
```

Comment:
Check `response.ok` here too?

Reply:
honestly i shouldn't even have touched this file, i just added the function to strip trailing slashes and added it here too because the fact you're not allowed to put a trailing slash on this particular env var has caused confusion in the past.

either way if the body isn't as expected the functions that call this endpoint (which is just a proxy to the multiplayer API) will fail parsing the response and that'll trigger error handling, so I think it's OK.